.travis.yml

```diff
@@ -24,5 +24,7 @@ matrix:
       script:
         - cd ui/
         - yarn install
+        - yarn eslint
+        - yarn stylelint
```

Comment:
IMHO `yarn stylelint` does not work

Reply:
It's in #10 . I want to avoid merge conflicts. I will reabase when #10 is on master.

Reply:
Ready for review.